src/datreant/core/tests/test_trees.py

```diff
@@ -234,15 +244,20 @@ def test_walk(self, tree, tmpdir):
             all_leaves = []
 
             for root, dirs, files in os.walk(tree.abspath):
+                # walk normally doesn't add slash to path, hacky fix
+                if root != tree.abspath:
+                    root = root + '/'
```

Comment:
See note below. Instead, use ``os.path.join(root, "")`` to add appropriate separator in a clean way.